chapters/05-funcoes-de-ordem-superior.md

```diff
@@ -30,13 +30,13 @@ Qual é mais propenso a erros?
 
 Se juntarmos as definições de `sum` e `range` o tamanho do segundo código será bem maior do que o primeiro, mas eu diria que ele é o mais correto.
```

Comment:
**Original:**
*"If we count the size of the definitions of sum and range, the second program is also big—even bigger than the first. But still, I’d argue that it is more likely to be correct."*

**Tradução:**
Se medirmos o tamanho das definições de `sum` e `range`, o segundo programa também será grande (até maior do que o primeiro). Mesmo assim, eu diria que ele é o mais provável a estar correto.